### PR TITLE
Feyzee LFG Fix

### PR DESCRIPTION
**Pull request by ghost**
Merge Feyzee's LFG Fix into trinity-wotlk-ai ; closes issue https://github.com/ike3/mangosbot/issues/19.
Enable TrinityCore Auction House Bot instead of ike3 AHbot.

Removed Rochet2 Transmog patch.
Reduced Feyzee's /src/ to 19 files instead of 2157,
thereby avoid reverting core scripts to their 2014 version.
(Feyzee never  changed any boss scripts, instances or zones.)

### CODE REVIEW

File: src/server/game/Chat/Chat.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2008-2015 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2008-2014 TrinityCore <http://www.trinitycore.org/>
```
Comment (Kittnz) [inline src/server/game/Chat/Chat.cpp]: Does not need to change

File: src/server/game/DungeonFinding/LFGQueue.cpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2008-2015 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2008-2014 TrinityCore <http://www.trinitycore.org/>
```
Comment (Kittnz) [inline src/server/game/DungeonFinding/LFGQueue.cpp]: Shouldn't change

Comment (ghost) [inline src/server/game/DungeonFinding/LFGQueue.cpp]: I know. I just was so tired when I finished this, after browsing through 1787 commits, that I fell asleep directly and did not wake up until now, couldn't think about it until now. :stuck_out_tongue_closed_eyes:

**Comment by Kittnz** [comment]
I haven't been able to compile with last pull from tc tho.

**Comment by ghost** [comment]
Hm, I wonder what may have changed. It was time-consuming enough to single out these 19 files and make the commit, but I also want to test and see if it is possible to merge again and then compile.
Edit; fun fact or trivia: https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Northrend/zone_wintergrasp.cpp still uses the header /\* Copyright (C) 2008 - 2009 Trinity http://www.trinitycore.org/ (https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Northrend/zone_wintergrasp.cpp#L1)

**Comment by Kittnz** [comment]
It's something to do with "format.h". 

https://github.com/TrinityCore/TrinityCore/commit/66ce97c87e347ab9426ffbd60b6fe5aa90a96b9b
maybe i can check it out again later..

**Comment by ghost** [comment]
OK, I will update the files with wrong header change, then test compiling it as it is, then try merging again with current TC and test compiling to see if it fails on format.h - I have seen that error before.

**Comment by ghost** [comment]
With this PR included, compiling in VS2013 works just as intended (79 warnings, though).
Win64 build:

```
22>------ Build started: Project: INSTALL, Configuration: Release x64 ------
22>  -- Install configuration: "Release"
22>  -- Installing: C:/Program Files/AI Playerbot/authserver.exe
22>  -- Installing: C:/Program Files/AI Playerbot/authserver.conf.dist
22>  -- Installing: C:/Program Files/AI Playerbot/worldserver.exe
22>  -- Installing: C:/Program Files/AI Playerbot/worldserver.conf.dist
22>  -- Installing: C:/Program Files/AI Playerbot/plugins.lib
22>  -- Installing: C:/Program Files/AI Playerbot/aiplayerbot.conf.dist.in
22>  -- Installing: C:/Program Files/AI Playerbot/ahbot.conf.dist.in
22>  -- Installing: C:/Program Files/AI Playerbot/mapextractor.exe
22>  -- Installing: C:/Program Files/AI Playerbot/vmap4assembler.exe
22>  -- Installing: C:/Program Files/AI Playerbot/vmap4extractor.exe
22>  -- Installing: C:/Program Files/AI Playerbot/mmaps_generator.exe
========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
```

Win32 build ; same as above, but with 81 warnings :

```
22>------ Build started: Project: INSTALL, Configuration: Release Win32 ------
22>  -- Install configuration: "Release"
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/authserver.exe
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/authserver.conf.dist
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/worldserver.exe
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/worldserver.conf.dist
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/plugins.lib
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/aiplayerbot.conf.dist.in
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/ahbot.conf.dist.in
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/mapextractor.exe
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/vmap4assembler.exe
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/vmap4extractor.exe
22>  -- Installing: C:/Program Files (x86)/AI Playerbot/mmaps_generator.exe
========== Build: 22 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
```

Not sure what accounts for the difference in number of warnings, but it compiles OK.

@Kittnz : I haven't tested any merging with newer source yet, but will do that at a later time.

**Comment by ghost** [comment]
Closing this PR because of too many commits that are already in the original source.
Will make another attempt at creating a PR at a later date if the Feyzee LFG fix is needed.

**Pull request closed without merging**